src/ResourceManager/Tags/Commands.Tags/Model/TagBaseCmdlet.cs

```diff
@@ -32,6 +32,9 @@ public TagsClient TagsClient
                         ErrorLogger = WriteErrorWithTimestamp
```

Comment:
@maddieclayton should these loggers be removed?

Reply:
done